docs/roles.md

```diff
@@ -31,21 +31,19 @@ The `owner` of hatvaults is HATs governance (currently a multisig, a DAO in the
 - setMaxBountyDelay - Set the timelock delay for setting the max bounty
 - can call `setFeeSetter` set the feeSetter role
 - can call `setCommittee` but only if the committee has not checked in yet
-- can call `setPoolInitialized` Set an "initialized" flag to disable "setShares"
-- can call `setShares` but only if the pool is not initialized
 
 ## `HatVaults - Vault committee` (committee set by each vault)
 
 - a specific address for each vault, typically a multisig
 - can call `committeeCheckIn`  to claim it's role - only after the committee has checked in, deposits to the vault are enabled
 - can call `setCommittee` - set new committee address. Can be called by existing committee if it had checked in, or by the governance otherwise.
 - can call `submitClaim` - submit a claim for a bounty payout
-- can call `setPoolWithdrawalFee` - set the fee for withdrawals from the pool
+- can call `setWithdrawalFee` - set the fee for withdrawals from the vault
```

Comment:
remove this line because it is only done by the feeSetter. and add:

- can call `setPendingMaxBounty` - set a pending request for the maximum percentage of the vault that can be paid out as a bounty
- can call `setMaxBounty` - set the vault's maximum bounty percentage to the already pending percentage

Reply:
Updated